cocoapods-trunk.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'nap', '>= 0.8', '< 2.0'
-  spec.add_dependency 'netrc', '0.7.8'
+  spec.add_dependency 'netrc', '0.10.0'
```

Comment:
please use a `~>` requirement for this again (it was exact because of a regression)